readme.md

```diff
@@ -1 +1,3 @@
 # public-sample
+
+1. add change
```

Comment:
why do you write on readme?

Reply:
this is the test repository!. anywhere is written!